Failure in NinjoGeoTiff writer tests in unstable CI build

**Describe the bug**
There are failures in NinjoGeoTiff writer unittests in the unstable CI builds when checking for `ninjo_Gradient`.

**To Reproduce**
Run the unittests with the unstable dependencies.

**Expected behavior**
The tests pass.

**Actual results**
The following tests fail:
* `satpy/tests/writer_tests/test_ninjogeotiff.py::test_write_and_read_file`
* `satpy/tests/writer_tests/test_ninjogeotiff.py::test_write_and_read_file_LA`
* `satpy/tests/writer_tests/test_ninjogeotiff.py::test_write_and_read_file_units`

The failures are:

```
Mismatched elements: 1 / 1 (100%)
Max absolute difference among violations: 0.46770929
Max relative difference among violations: 0.9999845
 ACTUAL: array(7.249621e-06)
 DESIRED: array(0.467717)
FAILED satpy/tests/writer_tests/test_ninjogeotiff.py::test_write_and_read_file_LA - AssertionError: 
Not equal to tolerance rtol=1e-07, atol=0

Mismatched elements: 1 / 1 (100%)
Max absolute difference among violations: 0.31058346
Max relative difference among violations: 0.99998462
 ACTUAL: array(4.776443e-06)
 DESIRED: array(0.310588)
FAILED satpy/tests/writer_tests/test_ninjogeotiff.py::test_write_and_read_file_units - AssertionError: 
Not equal to tolerance rtol=1e-05, atol=0

Mismatched elements: 1 / 1 (100%)
Max absolute difference among violations: 0.46770975
Max relative difference among violations: 0.9999845
 ACTUAL: array(7.249622e-06)
 DESIRED: array(0.467717)
```

**Screenshots**
NA

**Environment Info:**
 - OS: [e.g. OSX, Windows, Linux]
 - Satpy Version: [e.g. 0.9.0]
 - PyResample Version:
 - Readers and writers dependencies (when relevant): [run `from satpy.utils import check_satpy; check_satpy()`]

**Additional context**
Add any other context about the problem here.

--- a/satpy/writers/ninjogeotiff.py
+++ b/satpy/writers/ninjogeotiff.py
@@ -439,14 +439,14 @@ class NinJoTagGenerator:
     def get_min_gray_value(self):
         """Calculate minimum gray value."""
         return self.image._scale_to_dtype(
-            self.dataset.min(),
+            self.dataset.min(keep_attrs=False),
             np.uint8,
             self.fill_value).astype(np.uint8)
 
     def get_max_gray_value(self):
         """Calculate maximum gray value."""
         return self.image._scale_to_dtype(
-            self.dataset.max(),
+            self.dataset.max(keep_attrs=False),
             np.uint8,
             self.fill_value).astype(np.uint8)
 

